AcadeMeData/tests/test_client.py

```diff
@@ -1,6 +1,8 @@
+import pytest
 from AcadeMe import views
 
 
+@pytest.fixture
 def test_homepage(client):
     request = client.get('')

```

Comment:
This is wrong please refer review of the following PR #113 should fix it

Reply:
It doesn't pass the test without '@pytest.fixure' line. 
We add the same issue with PR #103.

Reply:
Added a test to make sure our query returns the correct results. :)

Reply:
@Yarboa

Reply:
It is not working with the `@pytest.fixture` decorator either. 
you just made it a fixture instead of a test, so it doesn't run when you run the test - what doesn't run cannot fail.

Reply:
@DanielItkes @DanWaldman please refer #113 and finish it, once @DanielItkes understand how to write a client test please propose a fix, sorry mate,

Reply:
Done, turns out I had problems with the test localy and by removing the cline test from the PR (which shouldn't have been included in the first place) it works now.

Reply:
You have added a search path('search/' dont you think you need a client test?